dataeng/resources/warehouse-transforms-ci.sh

```diff
@@ -69,9 +71,21 @@ fi
 if [ "$isApplications" == "true" ]
```

Comment:
I added these lines manually in the shell script of jenkins dsl, it ran fine over the weekend and I think we are good to merge this.